talent/job-descriptions/culture-section.md

```diff
@@ -1,11 +1,13 @@
 ## Culture 
 
-- source{d} is a company for developers by developers. We firmly believe in always doing what's best for the community, not necessarily companies. "For developers" is the answer to any strategic question or dilemma in the company. Always For Developers.
+- source{d} is a company for developers by developers. We firmly believe in always doing what's best for the individual developer in the community. Our team consists of members who are passionate about programming. To understand our culture better, read more about it [here](https://github.com/src-d/guide/blob/master/general/culture.md#for-developers).
 
-- "By developers" is our identity. Everyone in the company is either a developer or has a developer/engineering mindset. We don't hire anyone who doesn't fit into this culture, it's the only way to make sure everyone understands developers and can always defend their interest.
+- At the moment, we are 25+ people from 10 different countries working as a distributed organization. Some of our team members are based in the Madrid, Spain or San Francisco office, others work remotely from around the world (Portugal, Estonia, Russia, and others).
```

Comment:
Madrid, Spain or San Francisco sounds like a weird list ...
Everybody knows Madrid is in Spain 😄 

I would say :
> Madrid, Lisbon, and San Francisco offices